src/test/java/org/takes/http/BkBasicTest.java

```diff
@@ -132,7 +128,11 @@ public void exec(final URI home) throws IOException {
      * BkBasic produces headers with addresses without slashes.
      *
      * @throws IOException If some problem inside
+     * @todo #519:30min In org.takes.rq.RqHeaders.Base#map is skipping the
```

Comment:
@sebing No, keep it ignored. Create a new unit test for that specific bug and ignore it as well. In the puzzle instruct the resolver to remove `@Ignore` on _both_ tests after fixing the bug.

Is it clearer now?